services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/ApplicationConfig.java

```diff
@@ -349,6 +391,19 @@ public HttpEndpoint deviceHttpEndpoint() {
     @Scope("prototype")
     public HttpEndpoint credentialsHttpEndpoint() {
         return new DelegatingCredentialsManagementHttpEndpoint<CredentialsManagementService>(vertx(),
-                credentialsService());
+                                                                                             credentialsService());
```

Comment:
Could you please revert it back to the original version? The previous formatting was fine.

Reply:
reverted